examples/off_limb_enhance.py

```diff
@@ -0,0 +1,69 @@
+"""
+===========================
+Enhancing Off-limb emission
+===========================
+
+This example shows how to enhance emission above the limb.
+"""
+import numpy as np
+import astropy.units as u
```

Comment:
import ordering is not wrong but is inconsistent with the other examples where you have imported astropy as a separate block.